lib/extension/availability.ts

```diff
@@ -31,45 +38,103 @@ export default class Availability extends Extension {
             return utils.minutes(device.options.availability.timeout);
         }
 
-        const type = this.isActiveDevice(device) ? 'active' : 'passive';
+        return utils.minutes(this.isActiveDevice(device) ? settings.get().availability.active.timeout : settings.get().availability.passive.timeout);
+    }
+
+    private getMaxJitter(device: Device): number {
+        if (typeof device.options.availability === 'object' && device.options.availability?.max_jitter != null) {
```

Comment:
```suggestion
        if (typeof device.options.availability === 'object' && device.options.availability?.max_jitter !== undefined) {
```

Would that make sense here?